<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Media Player</string>
  </property>
  <property name="toolTip">
   <string>Open</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonOpenFile">
          <property name="toolTip">
           <string>Open</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="picsresource.qrc">
            <normaloff>:/pics/pics/6794469_document_file_folder_multimedia_music_icon.png</normaloff>:/pics/pics/6794469_document_file_folder_multimedia_music_icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonPlay">
          <property name="toolTip">
           <string>Play</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="picsresource.qrc">
            <normaloff>:/pics/pics/6794458_media_multimedia_music_play_player_icon.png</normaloff>:/pics/pics/6794458_media_multimedia_music_play_player_icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonPause">
          <property name="toolTip">
           <string>Pause</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="picsresource.qrc">
            <normaloff>:/pics/pics/6794465_audio_media_multimedia_music_pause_icon.png</normaloff>:/pics/pics/6794465_audio_media_multimedia_music_pause_icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStop">
          <property name="toolTip">
           <string>Stop</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="picsresource.qrc">
            <normaloff>:/pics/pics/6794471_media_multimedia_music_player_stop_icon.png</normaloff>:/pics/pics/6794471_media_multimedia_music_player_stop_icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonMute">
          <property name="toolTip">
           <string>Mute</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <activeoff>:/pics/pics/6794456_audio_multimedia_music_sound_speaker_icon.png</activeoff>
            <activeon>:/pics/pics/6794470_audio_media_multimedia_music_mute_icon.png</activeon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderVolume">
          <property name="toolTip">
           <string>Volume</string>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="picsresource.qrc"/>
 </resources>
 <connections/>
</ui>
